### PR TITLE
Firebase

### PR DESCRIPTION
**Pull request by ob1kenob1**
now we can get info from firebase and login with google)

### CODE REVIEW

File: src/app/core/services/firebase.service.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import { Injectable } from '@angular/core';
+import { AngularFirestore, AngularFirestoreCollection } from '@angular/fire/firestore';
+import { Observable } from 'rxjs';
+
+export interface Item { 
+  name?: string;
+  price?: number;
+  description?: string;
+}
```
Comment (TsirohOleh) [inline src/app/core/services/firebase.service.ts]: Create separate files for all interfaces.

File: src/app/modules/menu/menu.component.ts [hunk 2]
```diff
@@ -1,15 +1,18 @@
...
 
 @Component({
   selector: 'app-menu',
   templateUrl: './menu.component.html',
   styleUrls: ['./menu.component.scss']
 })
 export class MenuComponent implements OnInit {
-
-  constructor() { }
+  menu: any;
```
Comment (TsirohOleh) [inline src/app/modules/menu/menu.component.ts]: Create Munu interface for it. And set it as public.

File: src/app/modules/login/login.component.html [hunk 3]
```diff
@@ -0,0 +1,11 @@
+<p>
+  login works!
+</p>
```
Comment (TopOleh) [inline src/app/modules/login/login.component.html]: do we need this? :)

File: src/app/modules/menu/menu.component.html [hunk 4]
```diff
@@ -1,3 +1,10 @@
 <p>
   menu works!
 </p>
```
Comment (TopOleh) [inline src/app/modules/menu/menu.component.html]: And this? :)

File: src/app/core/interfaces/firebase-item.ts [hunk 5]
```diff
@@ -0,0 +1,7 @@
+export interface FirebaseItem {
+    name?: string,
+    price?: number,
+    weigth?: number,
```
Comment (TopOleh) [inline src/app/core/interfaces/firebase-item.ts]: Are all of these params optional?
Does it match our Dish interface or it is for another data?

File: src/app/core/services/firebase.service.ts [hunk 6]
```diff
@@ -0,0 +1,26 @@
+import { Injectable } from '@angular/core';
+import { AngularFirestore, AngularFirestoreCollection } from '@angular/fire/firestore';
+import { Observable } from 'rxjs';
+import { FirebaseItem } from 'app/core/interfaces/firebase-item'
+
+
```
Comment (TopOleh) [inline src/app/core/services/firebase.service.ts]: remove empty lines

File: src/app/core/services/firebase.service.ts [hunk 7]
```diff
@@ -0,0 +1,26 @@
+import { Injectable } from '@angular/core';
+import { AngularFirestore, AngularFirestoreCollection } from '@angular/fire/firestore';
+import { Observable } from 'rxjs';
+import { FirebaseItem } from 'app/core/interfaces/firebase-item'
```
Comment (TopOleh) [inline src/app/core/services/firebase.service.ts]: add semicolon

**Pull request merged**